ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/SandboxServer.scala

```diff
@@ -330,6 +338,10 @@ final class SandboxServer(
         ledgerType,
         authService.getClass.getSimpleName,
         config.seeding.fold("no")(_.toString.toLowerCase),
+        config.profileDir match {
+          case None => ""
+          case Some(profileDir) => s", profile directory = ${profileDir}"
+        },
```

Comment:
`fold` might be cleaner here.

```suggestion
        config.profileDir.fold("").fold(dir => s", profile directory = ${dir}")
```

Reply:
I'm not a big fan of `fold` and `foreach` on `Option`. At least for me, it makes reading code significantly harder since I need to run a type inference in my head, particularly during code review. And I don't mind writing a few more lines if it makes my intent easy to understand. It would have been cool if Scala had something like Rust's [`map_or`](https://doc.rust-lang.org/std/option/enum.Option.html#method.map_or).

Reply:
Fair. 🙂